### DISCUSSION TITLE
[Quality of Life Improvement] Automatically clear land when placing building

### DISCUSSION THREAD

**Discussion by fraang** [Keriew/augustus#315] [Ideas]
As far as I have seen there is no issue for an improvement like that. If there is, I am sorry.

A small (I hope at least) quality of life improvement would be to clear land when you place a building. It is bit tedious to first clear the land and then place the building. You could of course just clear an area but that might waste valuable trees. Especially on desert maps.

This might be interesting for julius as well.

**Comment by Areldir** [comment]
Just moved this over to the discussions board, seeing as it isn't really an issue. 

I believe there was some discussion of this feature before. Whilst the code exists in the base game, I'm not sure the idea progressed any further than discussion. @crudelios will have more details.

**Reply by Trass3r** [reply]
> the code exists in the base game

So it is already in the code somewhere, just dormant?

**Comment by crudelios** [comment]
We've been talking about this for about an year now.

This is a good idea and we will eventually get to it, it's just that no one started working on this yet.

**Comment by Trass3r** [comment]
Especially on Android this would be very beneficial given the imprecise touch input.
